src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/ov_plugin/core_integration.cpp

```diff
@@ -256,6 +256,78 @@ TEST(OVClassBasicTest, smoke_SetConfigEnableProfiling) {
     ASSERT_EQ(enableProfiling, value);
 }
 
+const auto bf16_if_supported       = InferenceEngine::with_cpu_x86_bfloat16() ?    ov::element::bf16 : ov::element::f32;
+const auto bf16_if_can_be_emulated = InferenceEngine::with_cpu_x86_avx512_core() ? ov::element::bf16 : ov::element::f32;
+using ExpectedModeAndType = std::pair<ov::hint::ExecutionMode, ov::element::Type>;
+
+const std::map<ov::hint::ExecutionMode, ExpectedModeAndType> exectedTypeByMode {
+    {ov::hint::ExecutionMode::UNDEFINED  , {ov::hint::ExecutionMode::PERFORMANCE,
+                                            bf16_if_supported}},
+    {ov::hint::ExecutionMode::PERFORMANCE, {ov::hint::ExecutionMode::PERFORMANCE,
+                                            bf16_if_supported}},
+    {ov::hint::ExecutionMode::ACCURACY,    {ov::hint::ExecutionMode::ACCURACY,
+                                            ov::element::f32}},
+};
+
+TEST(OVClassBasicTest, smoke_SetConfigExecutionModeExpectCorrespondingInferencePrecision) {
+    ov::Core ie;
+    const auto inference_precision_default = ov::element::f32;
```

Comment:
```suggestion
    const auto inference_precision_default = bf16_if_supported;
```

Reply:
Good catch, this test actually fails on spr